### PR TITLE
Local storage Unit test

### PR DESCRIPTION
**Pull request by Xclusivecyborg**
Wrote the unit test for local storage.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -49,6 +49,7 @@ dev_dependencies:
 
   flutter_test:
     sdk: flutter
+  test:
```
Comment (pauleke65) [inline pubspec.yaml:52]: why?
this is not required

File: test/services/local_storage_services_test.dart [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import 'package:hng/services/local_storage_services.dart';
+import 'package:hng/utilities/storage_keys.dart';
+import 'package:test/test.dart';
```
Comment (pauleke65) [inline test/services/local_storage_services_test.dart:3]: please remove package
We're using mockito

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Changes Requested

File: pubspec.yaml (hunk 1 above)
Comment (Xclusivecyborg) [reply pubspec.yaml]: Apologies about that. Changes have been made and pushed. Kindly check

**Pull request closed without merging**